augur/server.py

```diff
@@ -10,50 +10,40 @@
 import json
 import os
 import base64
+import logging
 
 from flask import Flask, request, Response, redirect
 from flask_cors import CORS
 import pandas as pd
 
 import augur
-from augur.util import logger
 from augur.routes import create_routes
 
 AUGUR_API_VERSION = 'api/unstable'
 
-class VueCompatibleFlask(Flask):
```

Comment:
@ccarterlandis : looks like we are removing a vue dependency from the server.py. duh ... curious how this works, or if it being in there was a legacy.

Reply:
It's legacy